lib/guardian.rb

```diff
@@ -541,6 +541,17 @@ def auth_token
     UserAuthToken.hash_token(token) if token
   end
 
+  def can_mention_here?
+    return false if SiteSetting.here_mention.blank?
+    return false if SiteSetting.max_here_mentioned < 1
+    return false if !authenticated?
+    return false if User.where(username_lower: SiteSetting.here_mention).exists?
+
+    return is_admin? if SiteSetting.min_trust_level_for_here_mention.to_s == 'admin'
```

Comment:
I think we have a bug here: 

```
[4] pry(main)> SiteSetting.min_trust_level_for_here_mention.to_s
=> "2"
[5] pry(main)> SiteSetting.min_trust_level_for_here_mention.to_i
=> 2
```

It seems like `SiteSetting.min_trust_level_for_here_mention` defaults to `"2"` so it'll never match `'admin'` or `'staff'`

Reply:
O my bad, the site setting is an enum and can be set to 'admin'`